components/stock-operation-modal/index.js

```diff
@@ -38,11 +38,20 @@ const StockOperationModal = (props) => {
   }, []);
 
   const getUserWallet = async () => {
+    const [apiResponse, setApiResponse] = useState(undefined);
```

Comment:
@Aryan-9211 On running it locally facing an error on this line. Please check
Reference  - https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

![image](https://user-images.githubusercontent.com/46373689/160255197-3810e6f5-db8d-4b91-8ee7-433d0ba7b666.png)